template.yml.erb

```diff
@@ -0,0 +1,543 @@
+AWSTemplateFormatVersion: 2010-09-09
```

Comment:
I think if we declare a constant array at the beginning of this template, we can use that in every template tag where we need the list of app types:

``` ruby
<%
  JAVALAB_APP_TYPES = %w(
   Theater
   Neighborhood
   Console
   Playground
  )
%>
```

Reply:
Or maybe declare one constant hash here instead with the list of app types, and store the configuration settings for each app type in the hash. We can then iterate over the keys in the hash when we just need the list of app types, and also access the specific Lambda config settings for each app type in the hash as well.

Reply:
👍 added the array for now. Will look into the hash as a follow-up.